Regression/RegressionTree/src/main/java/org/tribuo/regression/rtree/TrainTest.java

```diff
@@ -121,20 +124,20 @@ public static void main(String[] args) throws IOException {
         SparseTrainer<Regressor> trainer;
         switch (o.treeType) {
             case CART_INDEPENDENT:
-                if (o.fraction <= 0) {
-                    trainer = new CARTRegressionTrainer(o.depth,o.minChildWeight,0.0f, 1, false, impurity,
+                if (o.fraction == 0) {
```

Comment:
It's probably better to fix the default value of fraction to be 1.0, and then remove this if clause entirely.

Reply:
done